wand/cdefs/pixel_wand.py

```diff
@@ -58,10 +59,16 @@ def load(lib, IM_VERSION, IM_QUANTUM_DEPTH, IM_HDRI):
     is_im_6 = IM_VERSION < 0x700
     is_im_7 = IM_VERSION >= 0x700
 
+    # Check for IBM Z Systems.
+    if 's309x' == platform.machine():
```

Comment:
typo / flipped digits, should be 's390x'.
```suggestion
    if 's390x' == platform.machine():
```
for the sake of completeness, we could also cover the non-64-bit variant 's390' (i.e., without the 'x'), like this
```suggestion
    if platform.machine() in ['s390', 's390x']:
```

Reply:
Thanks!